https://issues.redhat.com/browse/ACM-9832 MSA default change docs (#6123)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/addon_managed_service.adoc

[#managed-serviceaccount-addon]
= Enabling ManagedServiceAccount add-ons

> When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you have to enable the add-on manually.

Reviewer: I think we should be explicit in noting the change in behavior.   So, I suggest that after the "When you install..." setence, we add a new one that reads  "This is a change from version 2.4 and earlier where ManagedServiceAccount add-on was disabled by default."

Author: Updated, thanks!

> When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. In {mce-short} version 2.4 and earlier, the `ManagedServiceAccount` add-on was disabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you have to enable the add-on manually.

Reviewer 2: From my perspective, it is unnecessary to mention this statement, and it's past tense: In {mce-short} version 2.4 and earlier, the `ManagedServiceAccount` add-on was disabled by default.

```suggestion
When you install the {mce-short} version 2.5 or later, the `ManagedServiceAccount` add-on is enabled by default. If you upgrade your hub cluster from {mce-short} version 2.4 and did not enable the `ManagedServiceAccount` add-on before upgrading, you must enable the add-on manually.
```

Author: Joe wanted us to have that sentence. Not sure how to proceed here.
